### PR TITLE
refactor: Move phone login to React auth pages

### PR DESCRIPTION
**Pull request by Yserz**

### CODE REVIEW

File: src/script/auth/page/CheckPassword.test.tsx [hunk 1]
```diff
@@ -0,0 +1,174 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
```
Comment (lipis) [inline src/script/auth/page/CheckPassword.test.tsx]:
```suggestion
 * Copyright (C) 2019 Wire Swiss GmbH
```

File: src/script/auth/page/PhoneLogin.test.tsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
```
Comment (lipis) [inline src/script/auth/page/PhoneLogin.test.tsx]:
```suggestion
 * Copyright (C) 2019 Wire Swiss GmbH
```

File: .elasticbeanstalk/config.yml [hunk 3]
```diff
@@ -3,6 +3,8 @@ branch-defaults:
     environment: wire-webapp-dev
   master:
     environment: wire-webapp-master
+  feat/phone-login:
```
Comment (bennycode) [inline .elasticbeanstalk/config.yml]: Can be removed before merging.

File: karma.conf.react.js [hunk 4]
```diff
@@ -35,9 +36,22 @@ module.exports = function(config) {
...
+    coverageIstanbulReporter: {
+      dir: path.resolve('docs/auth-coverage/'),
+      fixWebpackSourcePaths: true,
+      'report-config': {
+        html: {subdir: 'html'},
+      },
+      reports: ['html', 'lcovonly'],
+      thresholds: {
+        emitWarning: false,
+        global: {
```
Comment (bennycode) [inline karma.conf.react.js:48]: Have we had those thresholds already? If not we should decide in our team if we want to accept those.

File: src/script/auth/module/action/AuthAction.test.ts [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+    await store.dispatch(actionRoot.authAction.doLoginPlain({email, password, clientType: ClientType.PERMANENT}));
+    expect(store.getActions()).toEqual([AuthActionCreator.startLogin(), AuthActionCreator.successfulLogin()]);
+    expect(spies.setLocalStorage.calls.count()).toEqual(5);
+    expect(spies.setCookie.calls.count()).toEqual(1);
+    expect(spies.fetchSelf.calls.count()).toEqual(1);
+    expect(spies.generateClientPayload.calls.count()).toEqual(1);
+    expect(spies.doInitializeClient.calls.count()).toEqual(1);
+  });
+
+  it('authentication is successful with too many clients error', async () => {
```
Comment (bennycode) [inline src/script/auth/module/action/AuthAction.test.ts]: That's not a proper sentence.

File: src/script/auth/page/SetPassword.test.tsx [hunk 6]
```diff
@@ -0,0 +1,93 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
```
Comment (bennycode) [inline src/script/auth/page/SetPassword.test.tsx]:
```suggestion
 * Copyright (C) 2019 Wire Swiss GmbH
```

**Comment by bennycode** [comment]
As discussed with @mythsunwind we will merge it and make adustments on account-pages if needed.

**Pull request merged**